lib/events/cd-events.js

```diff
@@ -35,7 +35,7 @@ module.exports = function (options) {
     var locality = (args.req$ && args.req$.cookies.NG_TRANSLATE_LANG_KEY && args.req$.cookies.NG_TRANSLATE_LANG_KEY.replace(/\"/g, '')) || args.req$ && args.req$.headers['accept-language'];
     if (!locality) locality = 'en_US';
     locality = formatLocaleCode(locality);
-    var zenHostname = args.req$.headers.host || '127.0.0.1:8000';
```

Comment:
@david-cahill this is a variable initialization, don't think your fix will work.